src/server-code/admin-server.js

```diff
@@ -219,6 +221,32 @@ export default function runAdminServer(serverFalcorModel) {
     });
   }
 
+  app.post('/googlelogin', (req, res) => {
+    const token = req.body.data;
+    const endpoint = `https://www.googleapis.com/oauth2/v3/tokeninfo?id_token=${token}`;
```

Comment:
You should remember to actually use `encodeURIComponent` here as you don't know the contents of the token and there may be non alphabetical characters